recipes/daniele77-cli/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "2.1":
```

Comment:
https://github.com/daniele77/cli/releases/tag/v2.1.0

Reply:
@perseoGI 
Thanks for fixing my mistake so quickly!